### PR TITLE
Created a Seeder File and name it as UserSeeder

### PR DESCRIPTION
**Pull request by YANCZY**
The following files that has been added are:
**DatabaseSeeder.php** (database/seeders/DatabaseSeeder.php)
**UserSeeder.php** (database/seeders/UserSeeder.php)

In the UserSeeder I added a script to create a user with random details. And UserSeeder is being called in DatabaseSeeder.

### CODE REVIEW

**Review by MarJose123 — COMMENTED** [review]
missing `UserSeeder`

**Review by MarJose123 — CHANGES_REQUESTED** [review]
Add the missing `UserSeeder` Class

**Comment by YANCZY** [comment]
I have added the UserSeeder into the commit.

Please also see the following files I have added

RoleSeeder - For Roles and Permission
Kernel - Add a middleware package of spatie

Also in the password of user is password 123 and everything else is we're using faker library.

File: config/app.php
Comment (MarJose123) [inline config/app.php:1]: Not needed, please remove.

**Review by MarJose123 — CHANGES_REQUESTED** [review]
Please remove the `  Spatie\Permission\PermissionServiceProvider::class,` in the `config/app.php`

**Review by MarJose123 — APPROVED** [review]

**Pull request merged**